src/models/HyperwalletVerificationDocument.js

```diff
@@ -14,7 +14,7 @@ export default class HyperwalletVerificationDocument {
     CONSTRUCTOR_FIELDS = ["category", "type", "status", "country", "reasons", "createdOn", "uploadFiles"];
 
     constructor(properties) {
-        this.CONSTRUCTOR_FIELDS.forEach(key => {
+        this.CONSTRUCTOR_FIELDS.forEach((key) => {
```

Comment:
If this parentheses is closed on this line, where is the open parentheses for line 19? just wanted to make sure it is setup properly